Monika After Story/game/0utils.rpy

```diff
@@ -1,3 +1,275 @@
+python early:
+    import singleton
+    me = singleton.SingleInstance()
+
+python early in mas_logging:
+    import datetime
+    import logging
+    import os
+    import platform
+    import store
+
+    #Thanks python...
+    from logging import handlers as loghandlers
+
+    # log tags
+    LT_INFO = "info"
+    LT_WARN = "Warning! ;_;"
+    LT_ERROR = "!ERROR! T_T"
+
+    LT_MAP = {
+        logging.INFO: LT_INFO,
+        logging.WARN: LT_WARN,
+        logging.ERROR: LT_ERROR,
+    }
+
+    #Consts
+#    _MAS_LOG_FORMAT = logging.Formatter(
+#        fmt="[%(asctime)s] [%(levelname)s]: %(message)s",
+#        datefmt="%Y-%m-%d %H:%M:%S"
+#    )
+
+    class MASLogFormatter(logging.Formatter):
+        """
+        log formatter all other mas logs should extend if they want
+        custom functionality.
+
+        Features:
+            - uses our own log tags
+            - defaults format with time and level name
+        """
+
+        def __init__(self, fmt=None, datefmt=None):
+            if fmt is None:
+                fmt = "[%(asctime)s] [%(levelname)s]: %(message)s"
+            if datefmt is None:
+                datefmt = "%Y-%m-%d %H:%M:%S"
```

Comment:
I'm being picky here, but can these default `fmt` and `datefmt` be global constants or static vars?